front-end/src/pages/LogIn/LogInNavigate.ts

```diff
@@ -0,0 +1,26 @@
+import { USER_SERVICE_URL } from '@/constants/api';
+import axios from 'axios';
+
+export async function sendRequest(username: string, password: string) {
```

Comment:
api mình nên để trong folder api hay để chung folder page?

Reply:
@Mtoan12 tạo folder api đúng ko, mình bỏ vô đó được k á

Reply:
@vanduyquang100 @ThangLe1307 
folder api để chứa các module để xử lý các request trong các class: UserApi, TypeApi
Ví dụ: https://github.com/salesync-org/reference-project/blob/main/frontend/src/api/gradeApi.ts
-> Mục đích là để 1 request như get danh sách gì đó có thể đồng bộ Request Body, endpoint ở nhiều nới sử dụng khác nhau

Nếu giờ mình có thêm mock thì tạm thời tạo 1 thư mục mock ở trong thư mục api để tạo cái endpoint giả nhưng vẫn phải viết các hàm request api trong class như trên

Reply:
> @vanduyquang100 @ThangLe1307 folder api để chứa các module để xử lý các request trong các class: UserApi, TypeApi Ví dụ: https://github.com/salesync-org/reference-project/blob/main/frontend/src/api/gradeApi.ts -> Mục đích là để 1 request như get danh sách gì đó có thể đồng bộ Request Body, endpoint ở nhiều nới sử dụng khác nhau
> 
> Nếu giờ mình có thêm mock thì tạm thời tạo 1 thư mục mock ở trong thư mục api để tạo cái endpoint giả nhưng vẫn phải viết các hàm request api trong class như trên

mock thì mình đã có thư mục `mocks/api` rồi, nhưng ví dụ button log in gọi api call thì mình cấu trúc file handle cái đó như nào

Reply:
vậy cứ viết mấy cái mock trong mocks/api đi

Reply:
> vậy cứ viết mấy cái mock trong mocks/api đi

Nhưng mà mock là để viết handle request, còn handle kết quả mock trả về, hoặc sau này handle kết quả response thật thì viết ở đâu á